### DISCUSSION TITLE
How to clear queued transactions in tx_pool

### DISCUSSION THREAD

**Discussion by earth2378** [paradigmxyz/reth#13180] [Help] [answered]
Now I run Base node with Reth, however I found out that there are around 5000 queued transactions in my node.
I want to clear out these transactions. Any way to do this? For Geth, I found a post for delete `transaction.rlp` 
https://ethereum.stackexchange.com/questions/1774/geth-how-to-clear-queued-transactions
or set tx_pool.lifetime.

**Comment by mattsse** [comment]
the referenced answer mentions shutting down the node then deleting the rlp file
same thing is possible with reth, local txs (if any) are persisted to the datadir/txpool-transactions-backup.rlp file

**Reply by earth2378** [reply]
However, I didn't found the txpool-transactions-backup.rlp file, do I need to shutdown the node first?

**Reply by mattsse** [reply, marked as answer]
only locally submitted (rpc eth_sendrawtransaction) are persisted across restarts

if you're node doesn't have any local txs, nothing is persisted on shutdown
